pkg/clusteragent/orchestrator/stats.go

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
```

Comment:
We could potentially put orchestrator related packages under `+build orchestrator` but then we would need to do this, depending on usage, also for the status part.